services/storage/rpc_service.go

```diff
@@ -107,5 +148,14 @@ func (r *rpcService) Read(req *ReadRequest, stream Storage_ReadServer) error {
 
 	w.flushFrames()
 
+	span.SetTag("values", w.vc)
```

Comment:
Is this a counter? Maybe a name the `num_values` would be more clear?

Reply:
`num_values` is good